### PR TITLE
Add builds for windows

### PR DESCRIPTION
**Pull request by normanmaurer**
Motivation:

Let's also build on windows during PR validation

Modifications:

Add build on windows during PR

Result:

Validate that all also pass on windows

### CODE REVIEW

File: codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java [hunk 1]
```diff
@@ -188,6 +190,8 @@ public void testResponseDecompression() {
     @Test
     public void testResponseBrotliDecompression() throws Throwable {
         Brotli.ensureAvailability();
+        // Failing on windows atm
+        Assume.assumeFalse(PlatformDependent.isWindows());
```
Comment (normanmaurer) [inline codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: @slandelle @hyperxpro these two tests fail on windows... I wonder if the native code not works as expected on windows.

Comment (hyperxpro) [reply codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: Does the CI have Visual C++ redists?

Comment (normanmaurer) [reply codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: no idea... but the failures is really an assert failure:

```
2021-05-19T16:21:56.7287182Z [ERROR] Failures: 
2021-05-19T16:21:56.7289070Z [ERROR]   HttpContentDecoderTest.testResponseBrotliDecompression:207 Content-Length header should match uncompressed string's length expected:<97715> but was:<92212>
2021-05-19T16:21:56.7292065Z [ERROR]   HttpContentDecoderTest.testResponseChunksBrotliDecompression:250 Content-Length header should match uncompressed string's length expected:<97715> but was:<92212>
2021-05-19T16:21:56.7293833Z [INFO] 
2021-05-19T16:21:56.7294428Z [ERROR] Tests run: 855, Failures: 2, Errors: 0, Skipped: 0
```

Comment (hyperxpro) [reply codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: Alright, not a dependency issue. I will investigate this and later do PR for enabling Brotli4j on Windows CI. Is that fine?

Comment (normanmaurer) [reply codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: yep... I will just skip these on windows for now.

Comment (normanmaurer) [reply codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: @hyperxpro any idea yet ?

Comment (hyperxpro) [reply codec-http/src/test/java/io/netty/handler/codec/http/HttpContentDecoderTest.java:194]: Unfortunately no. It's working fine cross-platform for me.

@slandelle How you compressed that data?

**Comment by normanmaurer** [comment]
I merged this one and we can investigate as a followup @hyperxpro @slandelle

**Pull request merged**